OsmAnd/src/net/osmand/plus/plugins/odb/OBDTextWidget.kt

```diff
@@ -24,18 +26,20 @@ open class OBDTextWidget(
 	private val fieldType: OBDTypeWidget,
 	customId: String?,
 	widgetsPanel: WidgetsPanel?) :
-	SimpleWidget(mapActivity, widgetType, customId, widgetsPanel) {
+	SimpleWidget(mapActivity, widgetType, customId, widgetsPanel), OBDWidgetOptions {
 	private val plugin = PluginsHelper.getPlugin(VehicleMetricsPlugin::class.java)
```

Comment:
There is a new requirePlugin method, preferable for places that works only with it